### ISSUE TITLE
twisted/test/test_htb.py doesn't run in a Python 3 test environment

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8716]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8716|
|Type|enhancement|
|Created|2016-08-02 11:25:38Z|
|Branch|https://github.com/twisted/twisted/tree/8716-rodrigc-htb-py3|

If I try to run twisted/test/test_htb.py under Python 3 with:

tox -e py35-tests twisted/test/test_htb.py

it fails 

<details><summary>Searchable metadata</summary>

```
trac-id__8716 8716
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8716_rodrigc_htb_py3 8716-rodrigc-htb-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470137138478548 1470137138478548
changetime__1470654113831154 1470654113831154
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/471

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

changes looks good. Please merge

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 800217c8d1aaa097d3ef0efcb8982467d5262c67
```
#!CommitTicketReference repository="" revision="800217c8d1aaa097d3ef0efcb8982467d5262c67"
Merge pull request #471 from twisted/8716-rodrigc-htb-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8716

Add test_htb and test_pcp to Python 3 tests
```
